papaparse.js

```diff
@@ -990,7 +990,7 @@ License: MIT
 	function ParserHandle(_config)
 	{
 		// One goal is to minimize the use of regular expressions...
-		var FLOAT = /^\s*-?(\d*\.?\d+|\d+\.?\d*)(e[-+]?\d+)?\s*$/i;
```

Comment:
making the regex case sensitive will miss the following float 1.2E3. Please address this by updating the regex to allow for the exponent be either e or E